app/views/users/index.html.erb

```diff
@@ -1,10 +1,21 @@
 <section class="users-section">
   <ul class="users-list">
     <% @users.each do |user| %>
+      
       <li>
-        Name: <%= user.name %>
+        Name: 
         <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <%= link_to user.name,  user_path(user), class: 'profile-link' %>
+          <% if current_user.pending_friends.include?(user)%>
+            <h6>Pending Request</h6>
+          <% elsif current_user.friends.include?(user) %>
+            <h6>Already Friend</h6>
+          <% elsif current_user == user%>
```

Comment:
**REQUIRED**
Let's remove all the logic from the views. Views related logic should be placed in the helpers. 
[The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)